xrpackage-api.js

```diff
@@ -0,0 +1,557 @@
+import * as THREE from './xrpackage/three.module.js';
+import wbn from './xrpackage/wbn.js';
+import { GLTFLoader } from './xrpackage/GLTFLoader.js';
+import { VOXLoader } from './xrpackage/VOXLoader.js';
+import Avatar from './xrpackage/avatars/avatars.js';
+import utils from './xrpackage/utils.js';
+import { xrTypeLoaders } from './xrTypes.js';
+import { XRPackageEngine } from './xrpackage-engine.js';
+
+const { requestSw } = utils;
+export const apiHost = `https://ipfs.exokit.org/ipfs`;
```

Comment:
I'm not sure where or if this was ever `import`ed by other packages, but it might be safer to re-export this `export` from `xrpackage.js` like it used to be, just in case somebody ever did something like `import { apiHost } from 'xrpackage.js'`

Reply:
Hm I'm not sure if it was ever used externally; I think we can just not export it.